manifests/manifests.go

```diff
@@ -2,68 +2,73 @@ package manifests
 
 import (
 	"bytes"
+	"embed"
 	_ "embed"
 	"fmt"
+	"io/fs"
+	"path/filepath"
 	"text/template"
+)
 
-	"github.com/pkg/errors"
+const (
+	wegoManifestsDir = "wego-app"
 )
 
-var Manifests [][]byte
+var (
+	Manifests [][]byte
```

Comment:
It seems like we can remove this array now? It is being filled out in init but only with the AppCRD which is already public, so then we can access AppCRD directly instead?

Reply:
yep